PlaygroundService/Controllers/PlaygroundController.cs

```diff
@@ -22,6 +23,25 @@ public PlaygroundController(IClusterClient client, ILogger<PlaygroundController>
             _client = client;
             _logger = logger;
         }
+        
+        [HttpGet("templates")]
+        public async Task<IActionResult> GetTemplateConfig()
```

Comment:
Maybe to GetTemplateList would be more correct since you are ultimately getting a list of templates available, not the configuration of a template.